tests/proxy_unit_tests/test_proxy_server.py

```diff
@@ -2208,3 +2208,34 @@ def test_get_timeout_from_request():
     }
     timeout = LiteLLMProxyRequestSetup._get_timeout_from_request(headers)
     assert timeout == 90.5
+
+
```

Comment:
can you add this in tests/litellm/proxy/test_proxy_server.py 

we are moving to organizing our tests in that manner for contributor PRs